.forge-snapshots/BinCustomCurveHookTest#test_Swap_CustomCurve.snap

```diff
@@ -1 +1 @@
-156820
\ No newline at end of file
+142393
```

Comment:
big gas saving for this case when the delta negate each other -- so reserveOfApp do not even update at all